src/app/business/block/fileShare.component.ts

```diff
@@ -180,9 +182,31 @@ export class FileShareComponent implements OnInit{
             this.aclCreateShow = true;
             this.aclsItems = [0];
             this.createAclsFormGroup.reset();
+            this.getAcls(selectedFileShare);
+            this.showIpErrorMessage = [false];
         }
         this.selectedFileShare = selectedFileShare;
     }
+
+    getAcls(selectedFileShare){
+        this.checkAclsIp = [];
+        this.FileShareAclService.getFileShareAcl().subscribe((res)=>{
+            let str = res.json();
+            str.forEach(item=>{
+                if(item.fileshareId == selectedFileShare.id){
+                    this.checkAclsIp.push(item.accessTo);
+                }
+            })
+        })
+    }
+    userIpChnage(index){
```

Comment:
fix typo.

Reply:
I have just modified it. Please check it

Reply:
I still see that the function name is not changed.  
(keyup)="userIpChnage(i)" -> (keyup)="userIpChange(i)"